engine/src/witness/common/cf_parameters.rs

```diff
@@ -22,18 +26,26 @@ pub struct VaultSwapParameters {
 	pub refund_params: ChannelRefundParameters,
 	pub dca_params: Option<DcaParameters>,
 	pub boost_fee: Option<BasisPoints>,
```

Comment:
Unrelated to this PR but just noticed this is `Option<BasisPoints>` - should it not just be `BasisPoints`? We can save a byte 😅

Reply:
Originally when I wrote this my thought was that 1 byte makes no difference in any chain (not BTC as we don't use that there) and that it actually makes more sense to use None instead of boost of zero. However seeing that the extrinsic doesn't take an option it makes sense to remove that for consistency.
I'll change that, I can push it on top of this now that I'm also modifying the SDK.

Reply:
Also technically the option saves you one byte compared to using `BasisPoints` when not being used, which is actually for all swaps using this as we don't support boost for any of these chains (EVM/SOL).

Reply:
Or you could just use a single byte (`u8`) like we did for BTC 🙂